### PR TITLE
Remove MySQL config from database.yml

### PR DESCRIPTION
**Pull request by lukemorton**
Production and staging DB configuration should be read from DATABASE_URL

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -1,21 +1,11 @@
...
-  port: <%= ENV.fetch("DATABASE_PORT") %>
-  # socket: <%# ENV.fetch("DATABASE_SOCKET") %>
-
 sqlite: &sqlite
   adapter: sqlite3
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
 
 development:
-  <<: *mysql
```
Comment (StevenLeighton21) [inline config/database.yml]: Should we use this permanently? It's a quick fix to get around docker issues when jumping between the projects.

Comment (lukemorton) [reply config/database.yml]: If you ask me we should aim to get back to MySQL ASAP via Docker. Right now, we're all using SQLite in development and so lets update the app to reflect that.

File: config/database.yml [hunk 2]
```diff
@@ -24,10 +14,4 @@ test:
...
   database: db/test.sqlite3
 
-staging:
-  <<: *mysql
-  database: income_collection_staging
-
-production:
-  <<: *mysql
-  database: income_collection_production
+# production should be injected via DATABASE_URL
```
Comment (StevenLeighton21) [inline config/database.yml]: There's still a local database that stores the results of our priority calculations, I believe we shouldn't remove this? Or will it be configured via env _alongside_ the connection to UH?

Comment (lukemorton) [reply config/database.yml]: This is injected via ENV.

**Pull request merged**